Mathlib/Topology/MetricSpace/Katetov.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2024 Luigi Massacci. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luigi Massacci
+-/
+
+import Mathlib.Topology.MetricSpace.Basic
+import Mathlib.Topology.UniformSpace.Cauchy
+
+/-!
+# Katetov Maps
+
+In this file we define Katetov maps (i.e. one point extensions of a metric) and establish
+basic properties of their space. We embed a metric space in its space of Katetov maps via
+the Kuratowski embedding.
```

Comment:
Nothing else in this file mention "Kuratowski". Is it intentional?

Reply:
My bad, the thing is I have a big file with the construction of the Urysohn space (using Katetov maps) and I’m splitting it into pieces to have small PRs, that was a leftover from the larger file. I’ll remove it and add it back into the next PR : )